ci-operator/jobs/openshift/release/openshift-release-release-4.6-periodics.yaml

```diff
@@ -8274,6 +8274,7 @@ periodics:
       - --target=endurance-install
       - --input-hash=$(BUILD_ID)
       - --input-hash=$(JOB_NAME)
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
```

Comment:
`/etc/pull-secret` does not seem to be anything (looking at volumes & volume mounts)

Reply:
fixed, thanks